docs/_pages/releases.md

```diff
@@ -35,6 +35,7 @@ sidebar:
     * `BeLessOrEqualTo`: Use `BeLessThanOrEqualTo`
 * Removed the `DefaultValueFormatter.SpacesPerIndentionLevel` property which was added during the development of v6, but wasn't removed before the release of v6 - [#2281](https://github.com/fluentassertions/fluentassertions/pull/2281)
 * Dropped direct support for .NET Core 2.x and .NET Core 3.x - [#2302](https://github.com/fluentassertions/fluentassertions/pull/2302)
+* Add `ForCondition` to `IAssertionsScope` to support chaining `.ForCondition()` after `.Then` - [#2324](https://github.com/fluentassertions/fluentassertions/pull/2324)
```

Comment:
`ForCondition` -> `ForConstraint` and since this is only relevant for extensions, move it down the the sub-section "Breaking Changes (for extensions)" right below.

Reply:
haha lol :)
I had this same typo before in the code and I was wondering why it does not compile ;)